wgpu-hal/src/vulkan/conv.rs

```diff
@@ -34,6 +34,7 @@ impl super::PrivateCapabilities {
             Tf::Bgra8Unorm => F::B8G8R8A8_UNORM,
             Tf::Rgba8Uint => F::R8G8B8A8_UINT,
             Tf::Rgba8Sint => F::R8G8B8A8_SINT,
+            Tf::Rgb10a2Uint => F::A2B10G10R10_UINT_PACK32,
             Tf::Rgb10a2Unorm => F::A2B10G10R10_UNORM_PACK32,
```

Comment:
bit confused why these and not `A2R10G10B10_UINT_PACK32`/`A2R10G10B10_UNORM_PACK32`
but it's consistent what was talked about over at WebGPU repo

Reply:
Vulkan's packed formats are written in reverse (relative to the other ones) for some reason.
See https://registry.khronos.org/vulkan/specs/1.3-extensions/html/chap49.html#texel-block-size

Reply:
huh fascinating. Well guess it makes sense when one thinks about where the bits are (red bits are low bits)